src/ui/src/builder/BuilderJournal.vue

```diff
@@ -190,11 +276,11 @@ const sortedEntries = computed<Record<string, JournalEntry | null>>(() => {
 
 async function loadEntries() {
```

Comment:
I think this is a bit weird to have `loadEntries` doing parrallel call whith only one `loading/error` state.

Maybe that's fine for now, but you can consider using VueUse to split both

```ts
const journalReqPayload = computed(() => ({
	headers: {
		"Content-Type": "application/json",
		key_contains: "wf-journal-",
		skip_keys: Object.keys(rawEntries.value),
	},
}))
const { data: journalData, isFetching: isFetchingJournal  } = useFetch(convertAbsolutePathtoFullURL("/api/data/retrieve"))
  .post(journalReqPayload)
  .json()
```